lib/SILGen/SILGenStmt.cpp

```diff
@@ -761,6 +761,13 @@ void StmtEmitter::visitReturnStmt(ReturnStmt *S) {
 }
 
 void StmtEmitter::visitThrowStmt(ThrowStmt *S) {
+  if (SGF.getASTContext().LangOpts.ThrowsAsTraps) {
+    SGF.B.createBuiltin(S, SGF.SGM.getASTContext().getIdentifier("int_trap"),
```

Comment:
It's better to create a `cond_fail %one, "error thrown"`, because then - at least when debugging - the user can see the message in the debugger

Reply:
Fixed!